zip-0000.rst

```diff
@@ -163,51 +166,91 @@ SHOULD be added to the ZIP metadata indicating the original authorship
+- review this ZIP to make sure it reflects current practice,
+- update the owners of this ZIP,
+- review ZIP editor access to the `ZIPs repository <https://github.com/zcash/zips>`,
+- update the <zips@z.cash> email alias, and
+- invite new editors to the Zcash Community Forums, and the #zips channel on Discord.
+
+If it has been at least 12 months since the last ZIP editor change, the ZIP editors should:
+- review this ZIP to make sure it reflects current practice.
+
+
+ZIP Editor Responsibilities
+---------------------------
 
 The ZIP Editors subscribe to the `Zcash Community Forum.
 <https://forum.zcashcommunity.com/>`__
 
-For each new ZIP that comes in, a ZIP Editor confirms the following:
+Each new ZIP SHOULD be submitted as a "pull request" to the ZIPs git
+repository [#zips-repo]_. It SHOULD NOT contain a ZIP number unless
+one had already been assigned by the ZIP Editors. The pull request
+SHOULD initially be marked as a Draft. The ZIP content SHOULD be
+submitted in RST format. Generating HTML for a ZIP is optional.
+
+For each new ZIP that comes in, the ZIP Editors will:
 
 * Read the ZIP to check if it is ready: sound and complete. The ideas
   must make technical sense, even if they don't seem likely to be
   accepted.
 * The title should accurately describe the content.
-* The ZIP draft must have been sent to the Zcash Community Forum or as
+* The ZIP draft must have been sent to the Zcash Community Forum and as
   a PR to the ZIPs git repository [#zips-repo]_.
 * Motivation and backward compatibility (when applicable) must be
   addressed.
 * The licensing terms are acceptable for ZIPs.
 
-If the ZIP isn't ready, the editor will send it back to the Owners for
-revision, with specific instructions.
+Reviewing a ZIP
+---------------
+
+Any ZIP Editor can suggest changes to a ZIP. These suggestions are the
+opinion of the individual ZIP Editor. ZIP Editors are expected to perform
+independent technical and process review, regardless of contractural or
+any other relationships.
+
+ZIP Owners can clarify, modify, or decline suggestions from ZIP Editors.
 
-Once the ZIP is ready for the repository it SHOULD be submitted as a
-"pull request" to the ZIPs git repository [#zips-repo]_ where it may
-get further feedback. It SHOULD NOT contain a ZIP number unless one
-had already been assigned by the ZIP Editors. The pull request SHOULD
-initially be marked as a Draft.
+Typically, the ZIP Editors suggest changes in two phases:
+- `content review`: multiple ZIP Editors discuss the ZIP, and suggest
+  changes to the overall content. This is a "big picture" review.
+- `format review`: one or two ZIP Editors do a detailed review of the
+  structure and format of the ZIP. This ensures the ZIP is consistent
+  with other Zcash specifications.
 
-The ZIP Editors will:
+If the ZIP isn't ready, the Editors will send it back to the Owners for
+revision, with specific instructions. This decision is made by consensus
+among the current Editors.
```

Comment:
I can't remember if we discussed consensus for sending a ZIP back to the proposers.